cli/cli.hs

```diff
@@ -47,6 +47,7 @@ import EVM.Types qualified
 import EVM.UnitTest
 import EVM.Effects
 import EVM.Expr (maybeLitWordSimp, maybeLitAddrSimp)
+import EVM.Effects (Config(dumpEndStates))
```

Comment:
I think the bot is right, no? There is already `import EVM.Effects` two lines above.